src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -230,6 +234,10 @@ const InstancePage = (): JSX.Element => {
               tableLabel={t(
                 "openbridgeTempDictionary:processor.processorsListTable"
               )}
+              onPaginationChange={(): void => {}} // @TODO MGDOBR-673
```

Comment:
Sorry.. but what is "TODO"? The referenced JIRA is about plugging "...SE instance detail page to API" whereas these lines relate to pagination. Have I misunderstood something?

Reply:
In the instance detail page, there is the list of processors belonging to it, represented by a table: that table is paginated.